.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+
+💐
+
+/cc @zendesk/dingo
+
+### Description
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
+
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
+
+### Risks
+* [RUNTIME] Does this change touch the apps runtime?
```

Comment:
Will the public know what the apps runtime is? Can we reword this to `Can this change affect apps rendering for a user?`